test/acceptance/behave/components/ngsiv2/api_entry_point/retrieve_api_resource.feature

```diff
@@ -45,7 +45,7 @@ Feature: general operations in Context Broker using NGSI API v1 and v2
   Actions After the Feature:
   Setup: stop ContextBroker
 
-  @version
+  @version @skip
```

Comment:
Maybe it would be a good practice to include a comment about every test with `@skip` in order to explain why that test is skipped (in this case, a copy paste or your above response could be used to fill that comment).